src/run_context.rs

```diff
@@ -0,0 +1,45 @@
+use crate::relocatable::PyRelocatable;
+use pyo3::{pyclass, pymethods};
+
+#[pyclass]
+pub struct PyRunContext {
+    pub pc: PyRelocatable,
+    pub ap: PyRelocatable,
+    pub fp: PyRelocatable,
```

Comment:
It's probably more efficient to use real `Relocatable` here. Also they shouldn't be public, that's why we have `#[getter]`.

Reply:
Fixed.